src/sentry/integrations/services/repository/model.py

```diff
@@ -13,7 +13,7 @@ class RpcRepository(RpcModel):
     organization_id: int
     name: str
     external_id: str | None
-    config: dict[str, Any]
+    config: dict[str, Any]  # TODO(Gabe): Verify whether we want this to be in the repr
```

Comment:
This field currently contains just the name of the repository. It could get more use in the future, so skipping it in the repr seems reasonable to me.